examples/universal-cnf/vppagent/pkg/ucnf/ucnf_nse.go

```diff
@@ -40,7 +116,34 @@ func NewUcnfNse(configPath string, verify bool, backend config.UniversalCNFBacke
 
 	configuration := common.FromEnv()
 
-	pe := config.NewProcessEndpoints(backend, cnfConfig.Endpoints, configuration, ceAddons)
+	//add logic here
+	ipamAddress, ok := os.LookupEnv("IPAM_ADDRESS")
```

Comment:
per the helm chart comment, the IPAM server address config should be obtained from the `nseconfig` (configmap)